src/components/connectTelegram.tsx

```
@@ -1,259 +1,591 @@
+						"Don't delete this channel or you will lose all your files in https://yourtgcloud.vercel.app/",
+					broadcast: true
+				})
+			);
+
+			const result = res as Result;
+
+			if (result?.chats?.[0].id) {
+				return {
+					channelTitle,
+					id: result.chats?.[0].id,
+					accessHash: result.chats?.[0].accessHash!
```

Biome (2.3.13)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)